### PR TITLE
Update the docker compose command for the integration tests

### PR DESCRIPTION
**Pull request by leeturner**
This PR updates the github actions test run to use the newer version of the command

### CODE REVIEW

**Review by alexjrk — APPROVED** [review]

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -45,5 +45,5 @@ jobs:
       - name: Integration Tests
         run: |
           cd examples/
```
Comment (alexjrk) [inline .github/workflows/tests.yml]: Shouldn't we use `cd examples/intro/` here?

Comment (alexjrk) [reply .github/workflows/tests.yml]: And in `docker-compose.yml` also we should change the context from `../` to `../../`

Comment (leeturner) [reply .github/workflows/tests.yml]: Hey, yeah, I think you are correct.  I am not really that proficient with the python project at the moment so wasn't aware that these were broken.  Let me update them and see if that fixes the issue

**Comment by alexjrk** [comment]
Also, I think we have to change `- ..:/app/` to `- ../../:/app/` in volumes in docker-compose

File: examples/intro/docker-compose.yml [hunk 2]
```diff
@@ -13,17 +13,17 @@ services:
       - "host.docker.internal:host-gateway"
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
-      - ..:/app/
+      - ../../app/
```
Comment (alexjrk) [inline examples/intro/docker-compose.yml]:
```suggestion
      - ../../:/app/
```

**Comment by leeturner** [comment]
@alexjircan Many thanks for your support on this one

**Pull request merged**